modules/prebidServerBidAdapter/index.js

```diff
@@ -601,10 +614,20 @@ export const processPBSRequest = hook('sync', function (s2sBidRequest, bidReques
   }
 }, 'processPBSRequest');
 
+function shouldEmitAnalyticsEvent(s2sConfig, response) {
```

Comment:
The naming sounds confusing, like a function that returns a bool that allows to emit the "analytics" event. 
Can we try to find something better?
Proposition:  `allowedPbsEvents`, `getPbsEventsEmitStatus`, `emitPbsEvents`, something like that.